NitroxPatcher/Patches/Dynamic/NoCostConsoleCommand_OnConsoleCommand_bobthebuilder_Patch.cs

```diff
@@ -0,0 +1,24 @@
+using System.Reflection;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Prevents "bobthebuilder" command from enabling "fasthatch" and "fastgrow", since both commands are managed by the server
+/// </summary>
+public sealed partial class NoCostConsoleCommand_OnConsoleCommand_bobthebuilder_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((NoCostConsoleCommand t) => t.OnConsoleCommand_bobthebuilder(default));
+
+    public static void Prefix(out (bool, bool) __state)
+    {
+        __state = (NoCostConsoleCommand.main.fastHatchCheat, NoCostConsoleCommand.main.fastGrowCheat);
+        ErrorMessage.AddWarning("fastHatch and fastGrow cheats weren't enabled because they're shared cheat commands. Please enable them manually with their respective commands \"fastgrow\" and \"fasthatch\"");
```

Comment:
What about calling NoCostConsoleCommand_OnConsoleCommand_fastgrow_Patch.Prefix()? That way we at least try to enable them.